### ISSUE TITLE
Spack run issue

### ISSUE THREAD

**Issue by streeve** [ORNL/AdditiveFOAM#29]
Hi there, I installed AdditiveFOAM with Spack as instructed but am still not able to run the tutorial files. I keep getting something like "foamDirectory: command not found", shown in the screenshot below. I have put the right path to OpenFOAM lirary in the ALLrun file, not sure why this is still happenning. Is there any document anyone can provide to show how to setup the necessary file path? 

<img width="634" alt="image" src="https://github.com/ExascaleAM/AdditiveFOAM/assets/57839515/78d438e2-efb9-4711-b21d-c1e7df2a7046">

**Comment by streeve** [comment]
@guannant I moved your question to the primary repo

**Comment by streeve** [comment]
@colemanjs any ideas here?

**Comment by guannant** [comment]
So I tried installing OpenFOAM with spack first and then installed AdditiveFOAM. Those "reconstructPar, transformPoints" commands from OpenFOAM are still not working. I was able to initialize the program but was not able to go further with the "runLayers" file. I guess I want to ask what would be the best way to deploy AdditiveFOAM: Is the command "spack install AdditiveFOAM" sufficient to automatically handle the OpenFOAM installation, or do I need to install OpenFOAM first—either with spack or manually—before proceeding with AdditiveFOAM?
